event.go

```diff
@@ -77,10 +77,10 @@ func (ev *Event) Validate() error {
 		if rule.Protocol == "" {
 			return ErrSGRuleProtocolInvalid
 		}
-		if rule.FromPort < 1 || rule.FromPort > 65535 {
```

Comment:
Would be nice to have 0 and 65535 as constants. If you ever came back to this code, please move them to constants :)